### PR TITLE
Requesting PID code for WyoLum/VeloKey project

### PR DESCRIPTION
**Pull request by wyojustin**
Thanks for this service!  

Justin

### CODE REVIEW

**Comment by Arachnid** [comment]
Hi,

This looks good, but your project page is 404 - as are the images on the GitHub page!

**Comment by wyojustin** [comment]
Thank you Nick Johnson!  I've just created the projects page and updated
the links in the README.md file.

https://github.com/wyolum/VeloKey
http://wyolum.com/projects/velokey/

More to come as we continue the development.

Justin

On Sat, Feb 6, 2016 at 4:06 PM, Nick Johnson <notifications@github.com>
wrote:

> Hi,
>
> This looks good, but your project page is 404 - as are the images on the
> GitHub page!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pidcodes/pidcodes.github.com/pull/133#issuecomment-180863820>
> .
>



-- 
Justin
<justin@wyolum.com>

**Comment by wyojustin** [comment]
Looks like you fixed it all.

Best Regards,

Anool Mahidharia
Work <http://www.lumetron.com/>
DiY <http://www.wyolum.com/>
Hackaday <http://hackaday.com/about/>
HackerSpace <http://www.makersasylum.com>
Git <https://github.com/anool>
Cycling <http://cyclists.in/profile/AnoolJM>
Workouts <http://www.dailymile.com/people/anool>
Flickr <http://www.flickr.com/photos/anool>
Picasa <http://picasaweb.google.com/anool.m>

File: 1209/B01D/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: B01D
```
Comment (Arachnid) [inline 1209/B01D/index.md]: This needs to be "layout: pid".

Comment (wyojustin) [inline 1209/B01D/index.md]: I made the change as requested to "layout: pid".  Please let me know if you
need me to submit a new pull request.

Thanks,
Justin

On Sun, Feb 7, 2016 at 12:31 PM, Nick Johnson <notifications@github.com>
wrote:

> In 1209/B01D/index.md
> <https://github.com/pidcodes/pidcodes.github.com/pull/133#discussion_r52120879>
> :
>
> > @@ -0,0 +1,8 @@
> > +---
> > +layout: B01D
>
> This needs to be "layout: pid".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pidcodes/pidcodes.github.com/pull/133/files#r52120879>
> .
>



-- 
Justin
<justin@wyolum.com>

**Comment by wyojustin** [comment]
Thanks!  I saw our pid in the latest push.

Justin

**Pull request merged**